website/src/components/index-page/powerful-tools.tsx

```diff
@@ -0,0 +1,54 @@
+export function PowerFulTools() {
```

Comment:
The video here is cut off at the bottom and breaks the flow of rounded edges. 

<img width="1357" alt="Screenshot 2024-03-19 at 10 44 53" src="https://github.com/graphql/graphql.github.io/assets/35999252/ca10eb07-e60d-4b9b-a1ff-311008b8ab48">

Reply:
Also the second video, loops and displays a empty slide. 


https://github.com/graphql/graphql.github.io/assets/35999252/11492423-e74b-426e-a06e-f0c4657db2bd

Reply:
> The video here is cut off at the bottom and breaks the flow of rounded edges.
> 
> <img alt="Screenshot 2024-03-19 at 10 44 53" width="1357" src="https://private-user-images.githubusercontent.com/35999252/313909722-ca10eb07-e60d-4b9b-a1ff-311008b8ab48.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTExMDgyODAsIm5iZiI6MTcxMTEwNzk4MCwicGF0aCI6Ii8zNTk5OTI1Mi8zMTM5MDk3MjItY2ExMGViMDctZTYwZC00YjliLWExZmYtMzExMDA4YjhhYjQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDExNDYyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhNjRjMzI5OTE4OWU2MjdkMGYxYWRlZmM3OTIzM2Q2Yzc0MGY0YzQxNWQ0YTQ2ZjE4ZGJjYWI1MGJjN2E3MDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ztzWxF6ky0uWonfu6F6V3c29vMuvjyqohGq6Of-BWMI">

this is tricky to record because when you hover some link you have this popup 
<img width="313" alt="image" src="https://github.com/graphql/graphql.github.io/assets/7361780/b8eb86d2-8ce3-4bbf-b0a7-ffb9ef6f8d39">
any ideas how to hide it?